moveit_core/robot_model/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
+cmake_minimum_required(VERSION 3.5)
 set(MOVEIT_LIB_NAME moveit_robot_model)
 
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
```

Comment:
seems to me lines 4-11 should be states just once in the root CMakeLists.txt of moveit_core, not repeated in every submodule / subfolder. is there an issue with that?

Reply:
18edf88b3e85f2c377e0618b3105ac55d0a5ee1a